src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/OffenderService.kt

```diff
@@ -81,6 +83,41 @@ class OffenderService(
     )
   }
 
+  fun getOffenderSummariesByCrns(
+    crns: List<String>,
```

Comment:
Is it worth using a Set instead to filter out potential duplicates?

Reply:
Good plan! 👍

Reply:
Done